packages/react-scripts/config/webpack.config.prod.js

```diff
@@ -460,4 +425,9 @@ module.exports = {
     tls: 'empty',
     child_process: 'empty',
   },
+  // Turn off performance hints in production because we utilize
+  // our own hints via the FileSizeReporter
+  performance: {
```

Comment:
`performance: false` also disables the processing while `hints: false` only disables the warning

Reply:
We probably do want to disable processing entirely here.